docs_src/hardwaresetup.md

```diff
@@ -27,19 +28,8 @@ Click the links for corresponding set up instructions.
 
     3. [Set up the pipeline](./OVMS/pipelinesetup.md)
 
-=== "Intel® Xeon® Processor"
-    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
-
-    2. Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/)
-
-    3. [**Optional**] Install [Docker Compose v2](https://docs.docker.com/compose/), if using the docker compose feature
-    
-    4. Install [Git](https://git-scm.com/download/linux)
-
-    3. [Set up the pipeline](./OVMS/pipelinesetup.md)
-
 === "Intel® Data Center GPU Flex 140/170"
-    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
+    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://dgpu-docs.intel.com/driver/installation.html#ubuntu-install-steps).
 
     2. Update the [Kernel](https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html)
```

Comment:
For this step, let's either remove this or point to the installation steps above, [this page https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html](https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html) is deprecated

Reply:
ok. that page is deprecated too - the kernel install is for a really old kernel version 5.15 there... latest ubuntu 22.04 would install a 6.X kernel